pkg/operator/ceph/cluster/osd/config.go

```diff
@@ -68,7 +68,7 @@ func (c *Cluster) generateKeyring(osdID int) (string, error) {
 	return keyring, s.CreateOrUpdate(deploymentName, keyring)
 }
 
-func (c *Cluster) osdRunFlagTuningOnPVC(osdID int) error {
+func (c *Cluster) osdRunFlagSlowTuningOnPVC(osdID int) error {
```

Comment:
For consistency with the fast tuning, how about updating this method to use the command line params instead of the mon store?

Reply:
Done.